pkg/kubelet/types/constants.go

```diff
@@ -43,8 +43,8 @@ const (
 // entries here should be moved to staging/src/k8s.io.api/core/v1/types.go
 // once the feature managing the condition graduates to Beta.
 const (
-	// PodHasNetwork indicates networking has been configured successfully for the
+	// PodReadyToStartContainers indicates networking has been configured successfully for the
```

Comment:
nit: can the comment be updated here as well please - instead of `indicates networking has been configured ... IP address(es) assigned` how about `pod sandbox is successfully configured and the pod is ready to launch containers.`

Reply:
updated.